app/models/concerns/bulkrax/has_matchers.rb

```diff
@@ -28,28 +28,28 @@ def matcher(name, args = {})
 
     def add_metadata(node_name, node_content)
       field_to(node_name).each do |name|
-        next unless field_supported?(name)
-        matcher = self.class.matcher(name, mapping[name].symbolize_keys) if mapping[name]
-        multiple = multiple?(name)
+        matcher = self.class.matcher(name, mapping[name].symbolize_keys) if mapping[name] # the field matched to a pre parsed value in application_matcher.rb
+        multiple = multiple?(name) # the field has multiple values. e.g. ['a', 'b', 'c']
+        object = object_name(name) || false # the field is an object of key:value pairs. e.g. { obj: { key: value }}
+
+        next unless field_supported?(name) || (object && field_supported?(object))
+
+        if object
+          Rails.logger.info("Bulkrax Column automatically matched object #{node_name}, #{node_content}")
+          parsed_metadata[object] ||= {}
+        end
+
         if matcher
-          result = matcher.result(self, node_content)
-          if result
-            if multiple
-              parsed_metadata[name] ||= []
-              parsed_metadata[name] += Array.wrap(result)
-            else
-              parsed_metadata[name] = Array.wrap(result).join('; ')
-            end
-          end
+          matched_metadata?(matcher, multiple, name, node_content, object)
         # we didn't find a match, add by default
```

Comment:
@bkiahstroud looks like you wrote this line a year ago, but I'm unsure what it means. what gets added by default?

Reply:
[That was actually Julie](https://github.com/samvera-labs/bulkrax/commit/ce6c2cf4006667535204c1f07ddd7b342b5bc643#diff-34a20e0c575e2645054032a1f243165362c406c960228991a6cdd109acc43b11R45). My name is on the blame because I touched that line when doing rubocop fixes. Unfortunately, I'm not sure what it means

Reply:
ah. ok thanks!

Reply:
@orangewolf do you understand what this means? is it still relevant?

Reply:
I had to get in my time machine, but I believe that comment has just gotten lost - https://share.getcloudapp.com/rRujRRmQ it used to be in the top of the multiple section. Basically it said "we didn't find a matcher so now we fall down here". I don't think it is a particuarly valuable code comment and its especially bad after Kiah's rubocop work somehow took it a line too high

Reply:
Let's remove it if it isn't helpful.